modules/nf-core/salmon/index/tests/main.nf.test.snap

```diff
@@ -18,9 +18,21 @@
             ]
         ],
         "meta": {
-            "nf-test": "0.9.0",
-            "nextflow": "24.04.4"
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.2"
         },
-        "timestamp": "2024-10-18T10:01:03.89824494"
+        "timestamp": "2025-01-17T16:47:11.559333"
+    },
+    "versions_transcriptome_only": {
+        "content": [
+            [
+                "versions.yml:md5,85337fa0a286ea35073ee5260974e307"
```

Comment:
Is it correct there are no actual output files emitted here, just versions.yml?

Reply:
That's how the author wrote the tests: https://github.com/nf-core/modules/blob/9bc6b5ab78f92ead239b9d9b6420943c4367660b/modules/nf-core/salmon/index/tests/main.nf.test#L28

I guess the index isn't snapshottable.

Reply:
We should be able to snapshot the filenames at least